.gitlab-ci.yml

```diff
@@ -59,6 +61,14 @@ build-test:
       - submission/submission-webapp/build/libs/submission-webapp-1.0.0.jar
       - submission/submission-handlers/build/libs/submission-handlers-1.0.0.jar
 
-mysql-itest:
-  stage: mysql-itest
+mongo-nfs-itest:
+  stage: mongo-nfs-itest
+  script: gradle clean :submission:submission-webapp:itest -PenableFire=false --rerun-tasks --stacktrace
+
+mysql-fire-itest:
```

Comment:
Yes, maybe it's not worth it to add this case since MySQL is scheduled to be shut off. Mongo/Fire would be the default case, i.e. no parameters. All the other cases are accessories